src/content/capture/video-pc/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<!--
+ *  Copyright (c) 2016 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+-->
+<html>
+<head>
+
+  <meta charset="utf-8">
+  <meta name="description" content="WebRTC code samples">
+  <meta name="viewport" content="width=device-width, user-scalable=yes, initial-scale=1, maximum-scale=1">
+  <meta itemprop="description" content="Client-side WebRTC code samples">
+  <meta itemprop="image" content="../../../images/webrtc-icon-192x192.png">
+  <meta itemprop="name" content="WebRTC code samples">
+  <meta name="mobile-web-app-capable" content="yes">
+  <meta id="theme-color" name="theme-color" content="#ffffff">
+
+  <base target="_blank">
+
+  <title>Video to peer connection</title>
+
+  <link rel="icon" sizes="192x192" href="../../../images/webrtc-icon-192x192.png">
+  <link href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="../../../css/main.css" />
+  <link rel="stylesheet" href="css/main.css" />
+
+</head>
+
+<body>
+
+  <div id="container">
+
+    <h1><a href="//webrtc.github.io/samples/" title="WebRTC samples homepage">WebRTC samples</a> <span>Stream from a video to a peer connection</span></h1>
+
+    <video id="leftVideo" controls muted>
+      <!-- NB CORS: https://bugzilla.mozilla.org/show_bug.cgi?id=1177793 -->
+      <source src="../../../video/chrome.webm" type="video/webm" />
+      <source src="../../../video/chrome.mp4" type="video/mp4" />
+      <p>This browser does not support the video element.</p>
+    </video>
+
+    <video id="rightVideo" autoplay controls></video>
+
+    <p>This demo requires Firefox 43 or above, Chrome 51 or above, or Chrome 50 with <strong>Experimental Web Platform features</strong> enabled.</p>
+
+    <p>A stream is captured from the video on the left using the <code>captureStream()</code> method, and streamed via a peer connection to the video element on the right.</p>
+
+    <p>View the browser console to see logging.</p>
+
+    <p>Several variables are in global scope, so you can inspect them from the console: <code>localPeerConnection</code>, <code>remotePeerConnection</code> and <code>stream</code>.</p>
+
+    <p>For more information about RTCPeerConnection, see <a href="http://www.html5rocks.com/en/tutorials/webrtc/basics/" title="HTML5 Rocks article about WebRTC by Sam Dutton">Getting Started With WebRTC</a>.</p>
+
+    <a href="https://github.com/webrtc/samples/tree/gh-pages/src/content/capture/video-pc" title="View source for this page on GitHub" id="viewSource">View source on GitHub</a>
+
+  </div>
+
+  <script src="../../../js/adapter.js"></script>
```

Comment:
Used to shim webkitRTCPeerConnection/RTCPeerConnection (and maybe other stuff, haven't checked).